app/schemas/models/selectors/cinematic.js

```diff
@@ -90,14 +90,14 @@ const DEFAULT_THANGTYPE = () => ({
  * @param {Object} thangDefaults
  * @returns {Function} sets thangDefaults on the thang property for a character.
  */
-const setCharacterDefaults = ({ pos, scaleX, scaleY }) =>
+const setCharacterDefaults = ({ pos: { x, y }, scaleX, scaleY }) =>
   character => {
     if (!character) {
       return
     }
 
     const thang = character.thang || {}
-    thang.pos = _.merge(pos, (thang.pos || {}))
+    thang.pos = { ...{ x, y }, ...thang.pos }
```

Comment:
To confirm, the intended logic here is "if x or y are present on `pos` and `x` or `y` are missing from `thang.pos` add it to `thang.pos`?

Reply:
Also can thang.pos.x ever explicitly be undefined (vs not present)?  This will set x to undefined in this case?

Reply:
Great question. The intended logic is to have the `thang.pos.x` and `thang.pos.y` replace the default values if they exist.

It should never be explicitly undefined, just missing.